src/frontends/ir/tests/CMakeLists.txt

```diff
@@ -4,11 +4,18 @@
 
 set(TARGET_NAME ov_ir_frontend_tests)
 
+list(APPEND IR_TESTS_DEPENDENCIES openvino_ir_frontend openvino_intel_cpu_plugin)
+if (NOT UNIX OR NOT ENABLE_INTEL_CPU)
+    list(REMOVE_ITEM IR_TESTS_DEPENDENCIES openvino_intel_cpu_plugin)
```

Comment:
It will be better to add new dependencies instead of excluding:
```
if(ENABLE_INTEL_CPU)
    list(APPEND DEPENDENCIES openvino_intel_cpu_plugin)
    list(APPEND COMPILE_DEFINITIONS ENABLE_INTEL_CPU)
endif()
```

BTW why do you use CPU plugin for tests? Can you use template plugin?

Reply:
Thanks for suggestion! Anyway, CPU dependency is removed